src/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -460,14 +460,16 @@ export const CloudEnvironmentModal = ({
         () => {
           return {
             ...baseProps,
-            href:
-              app &&
-              (cloudProvider === cloudProviderTypes.AZURE
-                ? app?.proxyUrls['cbas-ui']
-                : app?.proxyUrls['cromwell-service']),
+            href: app && (cloudProvider === cloudProviderTypes.GCP ? app?.proxyUrls['cromwell-service'] : ''),
             onClick: () => {
               onDismiss();
               Metrics(signal).captureEvent(Events.applicationLaunch, { app: appTools.CROMWELL.label });
+              if (
+                cloudProvider === cloudProviderTypes.AZURE &&
+                Nav.getCurrentRoute().name !== 'workspace-workflows-app'
+              ) {
+                Nav.goToPath('workspace-workflows-app', { namespace, name });
```

Comment:
The route for the link should be in the `href` attribute, not in an `onClick` handler. Using `onClick` to navigate breaks some expected behavior: the links destination URL can't be seen in the status bar when hovering over the link, right click and open in new tab doesn't work, etc.

Reply:
thanks @nawatts. the reason I used the `onClick` handler is so that I could use `Nav.goToPath`, which has the desired behavior of not opening a new tab by default. is there a way for me to make that happen using the `href` attribute instead?

Reply:
Remove `Utils.newTabLinkPropsWithReferrer` from the Link props.